src/NUnitFramework/framework/Internal/Filters/NotFilter.cs

```diff
@@ -99,5 +99,13 @@ public override bool IsExplicitMatch(ITest test)
 
         //    return false;
         //}	
+
+        public override TNode AddToXml(TNode parentNode, bool recursive)
+        {
+            TNode result = parentNode.AddElement("not");
+            if (recursive)
+                BaseFilter.AddToXml(result, true);
+            return result;
+        }
     }
```

Comment:
I don't think so because the not element is combined with the parent element. Is that correct @CharliePoole?

@oznetmaster, when you code review, would you mind reviewing the overall pull request (from the files changed tab) rather than the individual commits? When you review the individual commits, your comments don't show up in the full list of changes making them difficult to see in relation to the final state of the code. It is just a shortcoming of GitHub that we need to deal with.